Libplanet/Blockchain/Policies/BlockPolicy.cs

```diff
@@ -129,6 +130,8 @@ public bool DoesTransactionFollowsPolicy(Transaction<T> transaction)
         }
 
         /// <inheritdoc/>
+        /// <exception cref="InvalidBlockStateRootHashException">It will be thrown when the
```

Comment:
How about move this to `IBlockPolicy<T>`?

Reply:
Other implementations don't have a duty to check the state root hash in this pull request. To become the duty, all blocks must need a state root hash.